tutorials/best_practices/godot_interfaces.rst

```diff
@@ -108,11 +108,13 @@ access.
         };
 
         // Warn users if the value hasn't been set.
-        public String _GetConfigurationWarnings()
+        public String[] _GetConfigurationWarnings()
```

Comment:
Since you are changing this, could you also use the language keyword instead of the framework type name?

```suggestion
        public string[] _GetConfigurationWarnings()
```